magellan-library/build.gradle

```diff
@@ -52,6 +52,8 @@ dependencies {
   testImplementation Dependencies.mockito
   testImplementation Dependencies.archTesting
   testImplementation Dependencies.robolectric
+
+  lintPublish project(':magellan-lint')
```

Comment:
This publishes the lint jar with the library so that we don't need a separate artifact.